helm/hpcc/templates/localroxie.yaml

```diff
@@ -63,10 +63,10 @@ spec:
 ---
 {{- range $service := $roxie.services }}
 {{- if ne (int $service.port)  0 }}
-{{- $name := printf "%s-%s" $roxie.name $service.name }}
 apiVersion: v1
 kind: Service
 metadata:
+  name: {{ $service.name | quote }}
   name: {{ $name | quote }}
```

Comment:
should this line been replaced?

Reply:
Seems to work either way. oddly. But yes it should